### DISCUSSION TITLE
Cannot install AS Review

### DISCUSSION THREAD

**Discussion by tombarnes52** [asreview/asreview#1536] [Troubleshooting (report errors in the "Issues" tracker)]
Hi

I am using a Mac with python installed 3.11.4. When trying to install AS review I get the following errors:

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for gevent
Failed to build gevent
ERROR: Could not build wheels for gevent, which is required to install pyproject.toml-based projects

Anyone able to assist? Thanks

**Comment by Rensvandeschoot** [comment]
Did you also get the error when using version 3.8 (see also https://github.com/asreview/asreview/issues/1533)

**Reply by tombarnes52** [reply]
thanks for reply. yes get the error for both versions

**Reply by Rensvandeschoot** [reply]
What version of gevent are you using?
